changelog/21.0/21.0.0/summary.md

```diff
@@ -24,6 +26,14 @@
     - **[vtctldclient ChangeTabletTags](#vtctldclient-changetablettags)**
     - **[Support for specifying expected primary in reparents](#reparents-expectedprimary)**
 
+## <a id="known-issue"/>Known Issue</a>
```

Comment:
```suggestion
## <a id="known-issues"/>Known Issue</a>
```

Reply:
id needs to be consistent between TOC and actual content.

Reply:
Fixed via [1205315](https://github.com/vitessio/vitess/pull/17062/commits/1205315b507ad45d65b6f6651e5ab47eb1c267f1)